src/workers/pool-number-refresh.js

```diff
@@ -0,0 +1,94 @@
+import Twilio from 'twilio'
+// import { r } from '../server/models'
+
+/**
+ * Get all numbers contacted in last 30 days.
+ */
+const getRecentContactNumbers = () => {
+  // TODO: replace this with database call after thinky to knex switch
+  return ['+13037171543', '+14145201377', '+14145348245']
+}
+
+/**
+ * Get just the area code from a given number.
+ */
+const getAreaCodeForNumber = (number) => number.replace(/[^0-9]/g, '').substring(0, 3)
+
+/**
+ * Put array of numbers in object keyed by area code.
+ */
+const groupNumbersByAreaCode = (numbers) => numbers.reduce((byAreaCode, number) => {
+  const areaCode = getAreaCodeForNumber(number)
+  if (!(areaCode in byAreaCode)) {
+    byAreaCode[areaCode] = []
+  }
+  byAreaCode[areaCode].push(number)
+  return byAreaCode
+}, {})
+
+/**
+ * Get numbers for Twilio poool from API.
+ */
+const getCurrentTwilioPoolNumbers = (twilio, sid) => {
+  const service = twilio.messaging
+    .services(sid)
+    .fetch()
+  return service
```

Comment:
when we return service, does that return an object that has available phone numbers in it?